financialaid/api_test.py

```diff
@@ -131,15 +133,22 @@ def setUpTestData(cls):
             user=cls.enrolled_profile2.user,
             tier_program=cls.tier_programs["25k"],
             status=FinancialAidStatus.PENDING_MANUAL_APPROVAL
-        )
-        # Country income thresholds
-        cls.country_income_threshold_0 = CountryIncomeThresholdFactory.create(
-            income_threshold=0
-        )
-        cls.country_income_threshold_50000 = CountryIncomeThresholdFactory.create(
+        )\
+        # Country income thresholds (these are already created via migrations, so just need to modify them)
+        cls.country_income_threshold_50000, _ = CountryIncomeThreshold.objects.get_or_create(
```

Comment:
It may not exist, since we're using `country=FuzzyText(length=2)` in the profile creation above.  I was thinking about using a hard-coded country code, but I felt this solution was more robust.